### PR TITLE
add line highlighting in custom model examples

### LINKED ISSUE #3815: On custom model and field names page, consider adding line highlighting in the code examples
IMO, the examples on https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/use-custom-model-and-field-names would be a bit easier to follow if we highlighted the relevant lines. For example, in the section https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/use-custom-model-and-field-names, the changed lines in the second code block could be highlighted.

### PR DESCRIPTION
**Pull request by jharrell**
Fixes #3815

### CODE REVIEW

File: content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/020-use-custom-model-and-field-names.mdx [hunk 1]
```diff
@@ -173,7 +173,7 @@ You can "rename" fields and models that are used in Prisma Client by mapping the
 
 _After_ you introspected your database with `prisma db pull`, you can manually adjust the resulting Prisma schema as follows:
 
-```prisma
+```prisma highlight=1-2;edit|4;edit|5-6;add|9;edit|12-13;edit|16;add|19-20;edit|25-26;edit|27-28;add|31-32;edit|35;edit|36-27;add|40-41;edit|44-45;edit|46-47;add
```
Comment (ruheni) [inline content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/020-use-custom-model-and-field-names.mdx:176]: The code snippet is too chaotic, and hard to follow the changes. I would suggest either:
- Using tabs — one tab containing the updates and the other one for the final schema (no highlights)
- A combo of a comment (right above the relation field) and highlight for the section with the `@@map("")` attribute function
- Using a simpler (our [standard](https://docs-git-jharrell-issue3815-prisma.vercel.app/about/prisma-docs/style-guide/schema-models)) schema with the Post & User models


https://docs-git-jharrell-issue3815-prisma.vercel.app/concepts/components/prisma-client/working-with-prismaclient/use-custom-model-and-field-names#using-map-and-map-to-rename-fields-and-models-in-the-prisma-client-api

CleanShot 2023-07-25 at 09 30 21@2x

**Comment by nikolasburk** [comment]
I'm not sure this PR is an improvement, it's very noisy indeed. At this point, I'd probably prefer not using any highlights and keep the code snippet plain. 

If you still feel strongly about line highlighting here, let's discuss on Slack how to best go about this 🙏
